scripts/merge_geojson/merge_geojson.py

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/env python
+"""
+Merge geojson files of the same type in the same directory, optionally to a
+newline delimited geojson file compliant with RFC 7464 for MapRoulette.
+"""
+import argparse
+import geojson
+import typing
+import glob
+import gzip
+import re
+import os
+import json
+import logging
+
+ASCII_RECORD_SEPARATOR = b"\x1E"
+ASCII_LINE_SEPARATOR = b"\x0A"
+
+
+def get_geojson_files(country_path: str, flag: str) -> typing.List[str]:
+    """
+    Get geojson files in a path for a flag
+    """
+    files = glob.glob(
+        os.path.join(country_path, "**", "*" + flag + "*"), recursive=True
+    )
+    return [g for g in files if re.search(r".*\.geojson(|\.gz)$", g)]
+
+
+def fix_suggestions_to_cooperativeWork(
```

Comment:
This is great, thanks for adding it.
We're working on bringing in a pyatlas-checks library to house some MapRoulette related code integrated with the [maproulette-python-client](https://github.com/osmlab/maproulette-python-client).

Once we have that checked in, we can repurpose this code within that cli.

Also.. check out this ticket: https://github.com/osmlab/maproulette-python-client/issues/63. If you have time, it would be awesome if you could additionally add a Cooperative Challenge model for the python client.